omod/src/main/webapp/resources/scripts/custom/visits.js

```diff
@@ -294,3 +294,10 @@ visit.buildVisitTypeAttributeParams = function(visitId){
 
     return params;
 }
+
+visit.setSelectedDropdownValue = function (id, element){
+    jq("#" + id).val(element.value);
+    if(element.name !== undefined){
```

Comment:
Same question here as above.
I would generally prefer a separate pull request for additional features. That way if they need to wait to be merged, it does not prevent the other fix from getting merged immediately.